android-auto-app/src/main/java/com/mapbox/maps/testapp/auto/car/CarCameraController.kt

```diff
@@ -85,6 +87,7 @@ class CarCameraController : MapboxCarMapObserver {
 
   override fun onScroll(
     mapboxCarMapSurface: MapboxCarMapSurface,
+    visibleCenter: ScreenCoordinate,
```

Comment:
do we need to add this parameter to the scroll interface? does it make sense to let it be the implementation detail?

Reply:
The visible center changes as part of the `onVisibleAreaChanged`. When we separated this interface from `MapboxCarMapObserver`, it simplifies the `DefaultMapboxCarMapGestureHandler`

If it is not a parameter, then we would also want `DefaultMapboxCarMapGestureHandler` to extend `MapboxCarMapObserver` so it can know the visible center.

The alternatives solutions have bigger issues IMO.

Reply:
Also considering it a valid `Mapbox` domain parameter for `onScroll` because `MapboxMap` has `dragStart(point: ScreenCoordinate)`.